hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -607,6 +610,38 @@ message ContainerBalancerStatusResponseProto {
   required bool isRunning = 1;
 }
 
+message ContainerBalancerStatusInfoRequestProto {
+  optional string traceID = 1;
+}
+
+message ContainerBalancerStatusInfoResponseProto {
+  required bool isRunning = 1;
```

Comment:
Adding all new fields as `optional` is recommended:

https://protobuf.dev/programming-guides/dos-donts/#add-required